pallets/creditcoin/src/lib.rs

```diff
@@ -236,6 +237,10 @@ pub mod pallet {
 	#[pallet::getter(fn gate_contract)]
 	pub type GATEConract<T: Config> = StorageValue<_, DeployedContract, ValueQuery>;
```

Comment:
Missed this typo in the last PR. Not sure whether fixing it should go in this PR or a separate one to keep this one clean, but let's not forget about it

Reply:
https://github.com/gluwa/creditcoin/pull/1281